ipapython/admintool.py

```diff
@@ -256,6 +259,7 @@ def _setup_logging(self, log_file_mode='w', no_file=False):
             log_file_name = self.options.log_file
         else:
             log_file_name = self.log_file_name
+        self.log_file_name = log_file_name
```

Comment:
This line should be moved in the `elif self.options.log_file:` block, otherwise the first call to _setup_logging(no_file=True) will reset self.log_file_name to None instead of keeping the default value /var/log/ipabackup.log

Reply:
You're right, nice catch.